### DISCUSSION TITLE
Deta Space 下线 | Deta Space Shut Down

### DISCUSSION THREAD

**Discussion by lizheming** [walinejs/waline#2704] [General]
Dear Deta Community,

[We've made the difficult decision to move on from Space](https://deta.space/sunset).

Space will keep running for 45 days until October 17, 2024. We will then start removing all apps (hosted or installed), horizons, websites and data.

You can continue to login until sunset. We've built tools to export your data (and code for developers).

More information on our [website](https://deta.space/sunset).

**Comment by CatIsNotFound** [comment]
一个不需要搭梯子且挺好用的平台，居然倒闭了/(ㄒoㄒ)/~~

---
A platform that does not require a ladder and is easy to use has closed down /(ㄒoㄒ)/~~

**Reply by CatIsNotFound** [reply]
纠正下，是停服了/(ㄒoㄒ)/~~

---
Correction, I stopped taking it/(ㄒoㄒ)/~~
